src/main/model/graph.ts

```diff
@@ -1,3 +1,4 @@
+import { node } from '../../test/defs/any.js';
```

Comment:
Look out for unintended changes, this is probably done by your editor (also note: imports from tests into main should not occur)